test/integration/patients/sidebar/action-sidebar.js

```diff
@@ -1065,9 +1176,9 @@ context('action sidebar', function() {
 
     cy
       .get('[data-activity-region]')
+      .find('[data-activities-region]')
       .should('contain', 'Clinician McTester (Nurse) added this Action from the Test Program program')
       .children()
-      .children()
```

Comment:
There's no longer an empty `<div>` wrapped around these, so the second `.children()` broke the test.